src/applications/static-pages/translation.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+import i18Content from './i18Content.json';
+import recordEvent from 'platform/monitoring/record-event';
+
+const configureTranslationLink = (e, currentLang, targetLang) => {
+  const contentDiv = document.getElementById('content');
+  contentDiv.lang = currentLang;
+  e.dataset.lang = currentLang;
+  e.innerText = i18Content[targetLang].linkTitle;
+  e.href = i18Content[targetLang].langToggleLink;
+  e.onclick = recordEvent({
+    event: 'faq-lang-toggle',
+    targetLang,
+  });
+};
+
+const displayTranslationLink = () => {
+  const i18LinkWrapper = document.getElementById('i18-link-wrapper');
+  if (!i18LinkWrapper) return;
+  const isSpanish =
+    window.location.href.includes('-esp') ||
+    window.location.href.includes('nodeId=14580');
+  const translatableLinks = [
+    // uncomment the below line once we get through staging review and remove references to preview node
+    // 'coronavirus-veteran-frequently-asked-questions',
+    'coronavirus-veteran-frequently-asked-questions-esp',
+    'nodeId=14580',
+  ];
+  const isTranslatable = translatableLinks.some(url =>
+    window.location.href.includes(url),
+  );
+
+  if (!isTranslatable) {
+    i18LinkWrapper.style.display = 'none';
+  }
```

Comment:
Should this be hidden by default and only display if the page is translatable? Just want to avoid a possible flash of content before the JS is downloaded and evaluated.

Reply:
good call, made the change.